mobile-app/app/screens/AppNavigator/screens/Loans/screens/PaybackLoanScreen.tsx

```diff
@@ -76,14 +76,20 @@ export function PaybackLoanScreen ({
     return new BigNumber(tokens.find((t) => t.id === id)?.amount ?? 0)
   }
 
+  const loanTokenOutstandingBal = new BigNumber(loanTokenAmount.amount)
+  const getAvailableLoanAmountToPay = (): string => {
+    const availableMaxPaybackAmt = BigNumber.min(loanTokenOutstandingBal, tokenBalance)
```

Comment:
The method name is pretty self explanatory, we can simplify to
```suggestion
    return BigNumber.min(loanTokenOutstandingBal, tokenBalance).toFixed(8)
```